src/pages-helpers/curriculum/docx/curriculum-download-patcher/patches/mainThreads.ts

```diff
@@ -1,119 +1,129 @@
-import { chunk } from "lodash";
+import { Element } from "xml-js";
 
 import { xmlElementToJson } from "../../xml";
 import { CombinedCurriculumData } from "..";
 
 import { createThreadOptions, threadUnitByYear } from "./util";
 
+import { Unit } from "@/components/CurriculumComponents/CurriculumVisualiser";
+
+function sortByOrder(units: Unit[]) {
```

Comment:
Can we move this to the [tab].tsx and pull it through? that way it's not being repeated in [tab]. It's a note for me for testing as well as I'm using it in the test file

Reply:
Going to leave this, I've added a ticket to next sprint to find a nice place for helper functions.